src/figuresClasses.ts

```diff
@@ -1,19 +1,84 @@
+type Color = 'red' | 'green' | 'blue';
+type Shapes = 'triangle' | 'circle' | 'rectangle';
```

Comment:
Better use enum here - it will give you more control and flexibility for Shapes type.